### PR TITLE
WIP Add loading spinner to the management console save action

### PR DESCRIPTION
**Pull request by chrismwendt**
This adds a spinner to the save action in the management console.

⚠️ WIP

Resolves https://github.com/sourcegraph/sourcegraph/issues/1822

### CODE REVIEW

**Comment by chrismwendt** [comment]
I get this error when running `bash serve.sh` from `cmd/management-console/web`:

```
  /Users/chrismwendt/github.com/sourcegraph/sourcegraph/cmd/management-console/web/src/CriticalConfigEditor.scss:1:1: Cannot resolve dependency '/Users/chrismwendt/github.com/sourcegraph/sourcegraph/cmd/management-console/web/node_modules/@sourcegraph/react-loading-spinner/lib/LoadingSpinner.css' at '/Users/chrismwendt/github.com/sourcegraph/sourcegraph/cmd/management-console/web/node_modules/@sourcegraph/react-loading-spinner/lib/LoadingSpinner.css'
    at Resolver.resolve (/Users/chrismwendt/github.com/sourcegraph/sourcegraph/cmd/management-console/web/node_modules/parcel-bundler/src/Resolver.js:70:17)
```

It looks like Parcel is saying a file that exists doesn't exist 😕 Any idea why?

File: dev/Procfile [hunk 1]
```diff
@@ -12,5 +12,4 @@ syntect_server: ./dev/syntect_server
 zoekt-indexserver: ./dev/zoekt-wrapper zoekt-sourcegraph-indexserver -sourcegraph_url http://localhost:3090 -index $HOME/.sourcegraph/zoekt/index -interval 1m -listen :6072
 zoekt-webserver: ./dev/zoekt-wrapper zoekt-webserver -index $HOME/.sourcegraph/zoekt/index -pprof -rpc
 management-console: PROJECT_ROOT=./cmd/management-console TLS_CERT=$HOME/.sourcegraph/management/cert.pem TLS_KEY=$HOME/.sourcegraph/management/key.pem management-console
-management-console-web: cd ./cmd/management-console/web && ./serve.sh
```
Comment (slimsag) [inline dev/Procfile]: shouldn't commit this

File: cmd/management-console/web/src/CriticalConfigEditor.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import '~@sourcegraph/react-loading-spinner/lib/LoadingSpinner.css';
```
Comment (slimsag) [inline cmd/management-console/web/src/CriticalConfigEditor.scss]: is this starting with tilde right?

File: dev/Procfile (hunk 1 above)
Comment (chrismwendt) [reply dev/Procfile]: Right, it's there to help repro the issue I'm seeing with Parcel. I'll remove it later.

**Comment by slimsag** [comment]
Will take a look locally soon.

File: cmd/management-console/web/src/CriticalConfigEditor.scss (hunk 2 above)
Comment (chrismwendt) [reply cmd/management-console/web/src/CriticalConfigEditor.scss]: Yes, I saw this elsewhere in our codebase, and I think it tells SCSS to look in `node_modules/`.

**Comment by slimsag** [comment]
Apologies, I ran out of time last night to take a look into this one. Maybe @attfarhan or @ijsnow might have quick ideas about why the import isn't working here?

**Comment by ijsnow** [comment]
It's possible that @attfarhan fixed this same sort of issue yesterday with bootstrap but I can't remember what the solution was off the top of my head.

**Comment by chrismwendt** [comment]
This won't get into 3.0, so there's less pressure to implement it soon.

**Pull request closed without merging**